torch/testing/_internal/opinfo/definitions/special.py

```diff
@@ -647,6 +658,20 @@ def sample_inputs_entr(op_info, device, dtype, requires_grad, **kwargs):
         ref=lambda x: scipy.special.spherical_jn(0, x) if TEST_SCIPY else None,
         supports_autograd=False,
     ),
+    UnaryUfuncInfo(
+        "special.harmonic_number",
+        decorators=(
+            toleranceOverride(
+                {
+                    torch.float32: tol(atol=1e-05, rtol=1e-05),
+                    torch.float64: tol(atol=1e-05, rtol=1e-05),
```

Comment:
This is very high for float64 no?

Reply:
This is because of my less than ideal ref.